doc/nrf/includes/sample_board_rows.txt

```diff
@@ -18,6 +18,10 @@
 
 | :ref:`nRF52840 Dongle <ug_nrf52>` | PCA10059 | :ref:`nrf52840dongle_nrf52840 <nrf52840dongle_nrf52840>` | ``nrf52840dongle_nrf52840`` |
 
+.. nrf21540dk_nrf52840
```

Comment:
This line is already in the file (last entry). Feel free to move, but please don't duplicate it, and don't link to the nRF52 user guide since there is no explanation about nRF21540 in there.

Reply:
I didn't realize that nRF21540 is already added in this file. I removed my entry